src/poll.rs

```diff
@@ -1289,6 +1289,42 @@ pub struct Iter<'a> {
     pos: usize,
 }
 
+/// Owned [`Events`] iterator.
+///
+/// This struct is created by the `into_iter` method on [`Events`].
+///
+/// # Examples
+///
+/// ```
+/// # use std::error::Error;
+/// # fn try_main() -> Result<(), Box<Error>> {
+/// use mio::{Events, Poll};
+/// use std::time::Duration;
+///
+/// let mut events = Events::with_capacity(1024);
+/// let poll = Poll::new()?;
+///
+/// // Register handles with `poll`
+///
+/// poll.poll(&mut events, Some(Duration::from_millis(100)))?;
+///
+/// for event in events.into_iter() {
```

Comment:
This could just be `for event in events` because Rust's [`for` loops are aware of `IntoIterator`](https://doc.rust-lang.org/std/iter/index.html#for-loops-and-intoiterator).